### DISCUSSION TITLE
Search for text between <...> in regexp, e.g. search for <var>

### DISCUSSION THREAD

**Discussion by ffasbend** [typst/typst#2511] [Q&A]
Hello, I wonder if it's possible to search for the characters < and > in regexp in raw text?
I tried 

#show raw: it => {
  show regex("for|in"): set text(weight:"bold")
  show regex("<(.+?)>"): set text(red, style:"italic")
  [#it]
}

but it did not match anything. 

if I replace [#it] with [#it].text it sees to work. Should it be done like that?

**Comment by Julian-Wassmann** [comment]
Hi,

The code you've posted works for me, even with just `[#it]`.

See the reproduction URL: https://typst.app/project/rtRTBikIwcTmYYVk5rfeB9

Maybe you tested your code in a more complex project? Can you share an example?
